packages/preferences/src/browser/preferences-contribution.ts

```diff
@@ -90,6 +95,12 @@ export class PreferencesContribution extends AbstractViewContribution<Preference
                 this.preferenceService.set(id, undefined, Number(this.scopeTracker.currentScope.scope), this.scopeTracker.currentScope.uri);
             }
         });
+        commands.registerCommand(PreferencesCommands.OPEN_USER_PREFERENCES, {
+            execute: async () => this.openPreferencesWithScope(PreferenceScope.User)
+        });
+        commands.registerCommand(PreferencesCommands.OPEN_WORKSPACE_PREFERENCES, {
+            execute: async () => this.openPreferencesWithScope(PreferenceScope.Workspace)
+        });
```

Comment:
To confirm, what happens when no workspace is present (ex: no workspace scope exists).
I believe in this case the command should not be present?